### DISCUSSION TITLE
Fitur Switch Dark Mode

### DISCUSSION THREAD

**Discussion by ghost** [upkoding/upkoding#23] [Ideas]
akan membuat pengguna lebih nyaman jika berselancar di website upkoding ketika dalam keadaan gelap.

**Comment by ekaputra07** [comment]
@gnwncpta hehe sabar ya, mau menyelesaikan fitur utama dulu (forum).

Atau kalau mau bantu mengimplementasikan dark mode, silahkan :) nanti saya bantu backendnya, tinggal dibantu mekanisme theme switchingnya aja.
